user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/Backups/Configuring_Elasticsearch_backups_Windows_Linux.md

```diff
@@ -312,3 +312,23 @@ You have now finished configuring an Elasticsearch backup. If you do not want to
 
 > [!IMPORTANT]
 > If you followed these steps in preparation of migrating your data from an existing Elasticsearch cluster to a new target Elasticsearch cluster, you can now run the migration as explained in [Running the migration with bespoke Elasticsearch data](xref:Migrating_the_general_database_to_a_DMS_Cassandra_cluster#running-the-migration).
+
+## Troubleshooting
+
+### Network share access in Windows
+
+Whether you are triggering the backup via DataMiner or using an Elasticsearch REST API client such as Kibana, you need to make sure that the backup path is accessible on all the nodes in the cluster.
+
+If the network path requires credentials, the SYSTEM user must be granted access. You can grant this access as follows:
+
+1. Follow the instructions under [Restrictions for the backup path](xref:Configuring_Elasticsearch_backups_Windows#restrictions-for-the-backup-path).
```

Comment:
Just to be sure, are line 324 and 326 indeed two steps that should be executed one after the other? Or are these in fact two different alternatives to grant the access? If it's the latter, could you adjust this so this is more clear, and perhaps also add why you would use one alternative instead of the other?

Reply:
They can use one of 2 options.